bitcoin/Cargo.toml

```diff
@@ -14,11 +14,13 @@ edition = "2018"
 
 # Please don't forget to add relevant features to docs.rs below
 [features]
-default = [ "std", "secp-recovery" ]
-rand = ["secp256k1/rand-std"]
-serde = ["actual-serde", "bitcoin_hashes/serde", "secp256k1/serde"]
+default = ["std"]
+crypto-std = ["secp256k1/std", "crypto"] # Enable this feature to use ECC (i.e. keys and sigs).
+crypto = ["secp256k1/alloc"]
```

Comment:
We should depend on `alloc` in `secp256k1` specification below and call the feature `secp256k1`. We only need `std` for `Error`, right?

Reply:
I called it `crypto` thinking that we were about move the `crypto` module to a crate and then this feature would enable stuff from that crate and from `secp25k1`, does that sound reasonable?

Reply:
Oh, OK then, if the crypto crate PR gets in the feature name should stay.